src/watchman_watcher.js

```diff
@@ -69,16 +69,18 @@ WatchmanWatcher.prototype.init = function() {
     return self.watchProjectInfo ? self.watchProjectInfo.root : self.root;
   }
 
-  function onVersion(error, resp) {
+  function onCapability(error, resp) {
     if (handleError(self, error)) {
+      // The Watchman watcher is unusable on this system, we cannot continue
+      process.exit(1);
```

Comment:
potentially mid next week:

For reference i believe the code is: https://github.com/ember-cli/ember-cli/blob/master/lib/models/watcher.js#L80-L125